renderer/pages/dashboard.tsx

```diff
@@ -43,9 +61,29 @@ export default function DashboardPage() {
 
   React.useEffect(() => {
     (async () => {
-      const data = await window.ipc.requestMetrics();
+      const data: Metrics = await window.ipc.requestMetrics();
+      console.log("requestMetrics", data);
+      if (data) {
+        dispatch(setMinorState(data));
```

Comment:
dispatch and setMinor state are missing in the dependency array of the useEffect. Also just do React.useEffect(async () =>
and don't do an anonymous function call, it can result in memory leak (async could do that as well, but that's a risk we must take)